tb_plugin/torch_tb_profiler/run.py

```diff
@@ -130,13 +131,22 @@ def get_gpu_metrics_data(profile):
             has_sm_efficiency = False
             has_occupancy = False
             is_first = True
+            gpu_info_columns = ["Name", "Memory", "Compute Capability"]
             for gpu_id in profile.gpu_ids:
                 if not is_first:
                     # Append separator line for beautiful to see.
-                    gpu_metrics_data.append({"title": "--------",
+                    gpu_metrics_data.append({"title": "<hr/>",
                                              "value": ""})
+
                 gpu_metrics_data.append({"title": "GPU {}:".format(gpu_id),
                                          "value": ""})
+                gpu_info = profile.gpu_infos.get(gpu_id, None)
```

Comment:
need we check the gpu_infos be null or not?

Reply:
Don't need in this case. Initialized as {} in run_generator.py